crates/bevy_diagnostic/src/basic_performance_diagnostics_plugin.rs

```diff
@@ -0,0 +1,104 @@
+use crate::{Diagnostic, DiagnosticId, Diagnostics};
+use bevy_app::{prelude::*, PluginGroupBuilder};
+use bevy_core::FrameCount;
+use bevy_ecs::prelude::*;
+use bevy_time::Time;
+
+/// This plugin group will add "frame time" diagnostics to an App, namely:
+/// * [`FrameTimeDiagnosticsPlugin`](crate::FrameTimeDiagnosticsPlugin)
+/// * [`FpsDiagnosticsPlugin`](crate::FpsDiagnosticsPlugin)
+/// * [`FrameCountDiagnosticsPlugin`](crate::FrameCountDiagnosticsPlugin)
+#[derive(Default)]
+pub struct BasicPerformanceDiagnosticsPlugins;
+
+impl PluginGroup for BasicPerformanceDiagnosticsPlugins {
+    fn build(self) -> bevy_app::PluginGroupBuilder {
+        PluginGroupBuilder::start::<Self>()
+            .add(FpsDiagnosticsPlugin)
+            .add(FrameTimeDiagnosticsPlugin)
+            .add(FrameCountDiagnosticsPlugin)
+    }
+}
+
+/// Adds "frame time" diagnostic to an App
+#[derive(Default)]
+pub struct FrameTimeDiagnosticsPlugin;
```

Comment:
We should avoid reusing this name; it'll lead to subtly broken "automatic" migration for users in the change over.

I would be okay with just naming these `FrameTimePlugin` and so on.

Reply:
Would the intend of an `FpsPlugin` be obvious to user?
How about `FpsMeasurePlugin` ?

Reply:
Yeah I'm fine with that. We can bikeshed the name(s) as we see fit: the key thing is avoiding the clash.

Reply:
Do agree that avoiding to use the names is a good idea though.
I'm simply not sure what the right names would be.
However, if you're confident about `FrameTimePlugin`, `FpsPlugin` and `FrameCountPlugin` I'll change them

Reply:
I think that's the simplest choice, and it's very easy to read the docs, especially with an IDE tooltip on hover.

Reply:
Seems like I didn't receive your messages in time. I'll revert my changes tomorrow :)

Reply:
Now renamed again to your suggestion `FrameTimePlugin`, `FpsPlugin` and `FrameCountPlugin`